### PR TITLE
first commit on branch gartee

### PR DESCRIPTION
**Pull request by gartee-john-PFG**
I did a super special change for you to check out

### CODE REVIEW

File: mergeExercise.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 line 1 from github
 line 2 from github
 line 3 from github
+line 4 from gartee
```
Comment (devinesydPFG) [inline mergeExercise.txt:4]: Please change this to upper case.

Comment (andereggkylePFG) [inline mergeExercise.txt:4]: Please change to upper case

**Review by andereggkylePFG — CHANGES_REQUESTED** [review]

File: mergeExercise.txt (hunk 1 above)
Comment (sanjaagic) [reply mergeExercise.txt:4]: I agree

**Comment by tatrokevinPFG** [comment]
Please make this change

**Review by sanjaagic — CHANGES_REQUESTED** [review]
What Kyle said.

**Review by tatrokevinPFG — CHANGES_REQUESTED** [review]
I agree

File: mergeExercise.txt (hunk 1 above)
Comment (gartee-john-PFG) [reply mergeExercise.txt]: fixed

**Review by andereggkylePFG — APPROVED** [review]
Beautiful!

**Review by sanjaagic — APPROVED** [review]

**Review by devinesydPFG — APPROVED** [review]

**Review by tatrokevinPFG — APPROVED** [review]

**Pull request merged**